### ISSUE TITLE
[Bug] [1.20.4 Fabric Server] Attempting to run a Minecraft command from Discord results in an exception

### ISSUE THREAD

**Issue by ArkoSammy12** [hypherionmc/sdlink#106]
Here's the full exception

```java
[19:34:59] [pool-12-thread-1/ERROR]: One of the EventListeners had an uncaught exception
java.lang.NoSuchMethodError: 'int net.minecraft.server.command.CommandManager.executeWithPrefix(net.minecraft.server.command.ServerCommandSource, java.lang.String)'
        at fabric.com.hypherionmc.sdlink.FabricPlatformHelper.executeCommand(FabricPlatformHelper.java:23) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at fabric.com.hypherionmc.sdlink.server.SDLinkMinecraftBridge.executeMinecraftCommand(SDLinkMinecraftBridge.java:153) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at fabric.com.hypherionmc.sdlink.core.discord.hooks.MinecraftCommandHook.discordMessageEvent(MinecraftCommandHook.java:55) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at fabric.com.hypherionmc.sdlink.core.discord.events.DiscordEventHandler.onMessageReceived(DiscordEventHandler.java:63) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at com.hypherionmc.sdlink.shaded.dv8tion.jda.api.hooks.ListenerAdapter.onEvent(ListenerAdapter.java:446) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at com.hypherionmc.sdlink.shaded.dv8tion.jda.api.hooks.InterfacedEventManager.handle(InterfacedEventManager.java:96) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at fabric.com.hypherionmc.sdlink.core.util.ThreadedEventManager.lambda$handle$0(ThreadedEventManager.java:20) ~[SimpleDiscordLink-Combo-1.20-3.0.1.jar:?]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539) ~[?:?]
        at java.util.concurrent.FutureTask.run(FutureTask.java:264) ~[?:?]
        at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:840) ~[?:?]
```

**Comment by hypherionmc** [comment]
Oh yeah, this is the one you reported in Discord.

I haven't forgotten about this. I've been working on a new system for CraterLib so our mods can run independently from Minecraft.

Actually showed off the first working implementation of this running Simple RPC to our team over the weekend. 

So yes, I am still going to fix this, but with it will also come improvements to prevent this from happening on future versions.

**Comment by ArkoSammy12** [comment]
> Oh yeah, this is the one you reported in Discord.
> 
> I haven't forgotten about this. I've been working on a new system for CraterLib so our mods can run independently from Minecraft.
> 
> Actually showed off the first working implementation of this running Simple RPC to our team over the weekend.
> 
> So yes, I am still going to fix this, but with it will also come improvements to prevent this from happening on future versions.

Sounds good, thank you. I just wanted to also have the issue here as I figured having the current list of issues to work on in Github is more convenient than in Discord. Thank you for answering.

**Comment by hypherionmc** [comment]
It definitely is more convenient, cause I forget stuff reported in Discord. Not even going to lie about it.

Can see you're a developer yourself xD

**Comment by ArkoSammy12** [comment]
> It definitely is more convenient, cause I forget stuff reported in Discord. Not even going to lie about it.
> 
> Can see you're a developer yourself xD

Ye Im a fellow modder xd

**Comment by blackdllx** [comment]
where can i download dev version? only found beta

**Comment by hypherionmc** [comment]
Finally fixed in 3.1.0

**Comment by ArkoSammy12** [comment]
> Finally fixed in 3.1.0

Awesome, thank you man 👍
